corehq/apps/data_interfaces/templates/data_interfaces/interfaces/case_management.html

```diff
@@ -80,17 +80,16 @@
     <div id="case-management">
         <div class="row">
             <form class="well form-inline" style="margin: 1em; display: none;" data-bind="submit: updateCaseOwners, caseReassignmentForm: selected_cases">
-                    <label for="reassign_owner_select" class="inline">{% trans 'Reassign selected cases to' %} </label>
-                    <span data-bind="visible: should_show_owners">
-                        <input type="hidden" name="reassign_owner"
-                            id="reassign_owner_select"
-                            style="width: 300px;"
-                            data-bind="grabUniqueDefault: selected_owners,
-                                       optionsText: 'name',
-                                       optionsValue: 'ownerid',
-                                       optionsCaption: '{% trans 'Select New Owner...' %}',
-                                       event: {change: changeNewOwner}, ic"></select>
-                    </span>
+                <label for="reassign_owner_select" class="inline">{% trans 'Reassign selected cases to' %} </label>
+                <span data-bind="visible: should_show_owners">
+                    <input type="hidden" name="reassign_owner"
+                        id="reassign_owner_select"
+                        style="width: 300px;"
+                        data-bind="optionsText: 'name',
+                                   optionsValue: 'ownerid',
+                                   optionsCaption: '{% trans 'Select New Owner...' %}',
+                                   event: {change: changeNewOwner}, ic" />
+                </span>
                 <button type="submit"
                         data-bind="hqbSubmitReady: is_submit_enabled, visible: is_submit_enabled"
```

Comment:
Is it possible to remove this double button definition? I think we could just remove the `visible` riht?

Reply:
I don't think so. From the blame it looks like biyeun spent a bit of time trying to get this behavior right, and from some brief toying with it I couldn't get it quite right, so I'm going to leave it as is